docker-compose.yml

```diff
@@ -42,6 +42,20 @@ services:
     depends_on:
       - elasticsearch
 
+  curator:
```

Comment:
Please move this service to https://github.com/deviantony/docker-elk/blob/master/extensions/.